charts/alfresco-transform-service/values.yaml

```diff
@@ -0,0 +1,316 @@
+repository:
+  edition: "Enterprise"
+transformrouter:
+  replicaCount: 2
+  nodeSelector: {}
+  image:
+    repository: quay.io/alfresco/alfresco-transform-router
+    tag: 2.0.0
+    pullPolicy: IfNotPresent
+    internalPort: 8095
+  service:
+    name: transform-router
+    type: ClusterIP
+    externalPort: 80
+  environment:
+    JAVA_OPTS: >-
+      -XX:MinRAMPercentage=50
+      -XX:MaxRAMPercentage=80
+  readinessProbe:
+    initialDelaySeconds: 140
+    periodSeconds: 60
+    timeoutSeconds: 10
+  livenessProbe:
+    initialDelaySeconds: 140
+    periodSeconds: 120
+    timeoutSeconds: 60
+  podSecurityContext:
+    runAsNonRoot: true
+    runAsUser: 33016
+  resources:
+    requests:
+      cpu: "0.25"
+      memory: "300Mi"
+    limits:
+      cpu: "1"
+      memory: "512Mi"
+pdfrenderer:
+  replicaCount: 2
+  nodeSelector: {}
+  image:
+    repository: alfresco/alfresco-pdf-renderer
```

Comment:
I think we should point to quay by default but we can tackle as pat of a different ticket if you prefer